docker-compose-flower.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3.8"
+
```

Comment:
this file was removed in my branch, i think yours adds it back. It was "replaced" with docker-compose-flower_rabbitmq.yaml